8BallWithoutActualBall/UserDefaultsValues.swift

```diff
@@ -7,21 +7,25 @@
 
 import Foundation
 
-class UserDefaultsValues {
+protocol DefaultValues {
+    var answer: String {mutating get set}
```

Comment:
после и перед фигурными скобками нужны пробелы
```suggestion
    var answer: String { mutating get set }
```